src/__tests__/ReduxRouter-test.js

```diff
@@ -223,8 +223,6 @@ describe('<ReduxRouter>', () => {
       'or a class/function (for composite components) but got: object. ' +
       'Check the render method of `ReduxRouterContext`.';
 
-    const routingContextInvalidPropErrorMessage = 'Invalid prop `RoutingContext` of type `object` supplied to `ReduxRouterContext`';
```

Comment:
@Scarysize I meant what was the error message in your terminal when running tests before you removed the lines